Assets/StartingPositionGenerator.cs

```diff
@@ -112,7 +112,7 @@ private void selectNextEnvironment()
         script.objectAmount = environmentConfiguration.GetNumberObjectsConfigValue();
         script.createNewEnvironment();
 
-        startingPosition = script.getMeshStartingVertex() + new Vector3(0.0f, 1.0f, 0.0f);
+        startingPosition = script.getMeshStartingVertex() + new Vector3(30.0f, 1.0f, -20.0f);
```

Comment:
This does not work

Reply:
The method gets the height of the MeshStartingVertex which is not the same as the height of that vertex x+30 and z-20

Reply:
In other words, you'll fall through the floor sometimes